modules/ImageClassifierService/app/predict.py

```diff
@@ -25,7 +24,7 @@
 
 def initialize():
     print('Loading model...',end=''),
-    with tf.gfile.FastGFile(filename, 'rb') as f:
```

Comment:
Thanks a lot for your contribution. Why are you replacing FastGFile by GFile? The official sample seems to be using FastGFile

Reply:
That was actually an unrelated change I put in to resolve a deprecation warning I was getting. I have amended my PR to remove this change.